src/plopp/graphics/canvas3d.py

```diff
@@ -5,13 +5,29 @@
 from .outline import Outline
 
 from copy import copy
-import numpy as np
 from ipywidgets import VBox
+import numpy as np
+from scipp import Variable
+from typing import Any, Tuple
 
 
 class Canvas3d(View, VBox):
-
-    def __init__(self, figsize=None, title=None):
+    """
+    WebGL canvas used to render 3D graphics.
+    It is based on the ``pythreejs`` library. It provides a scene, camera, controls,
+    as well as functions for controlling the camera.
+    In addition, it draws a RGB axes helper for the XYZ dimensions, and can draw an
+    outline box, given a spatial range in the three directions.
+
+    Parameters
+    ----------
+    figsize:
+        The width and height of the renderer in pixels.
+    title:
+        The title to be placed above the figure.
+    """
+
+    def __init__(self, figsize: Tuple[float, float] = None, title: str = None):
```

Comment:
Doesn't call super's `__init__`, intentional?

Reply:
Nice catch, this doesn't need to inherit from anything actually. Thanks!

Reply:
Thank PyCharm. I just followed looked at its warnings :)